README.md

```diff
@@ -56,81 +56,104 @@ You can then easily use Klaviyo to track events or identify people.  Note, track
-    client.list_subscription(list_id, subscription_type, data=data, method="GET")
-    
-    # you can unsubscribe customers from a list
-    client.unsubscribe_from_list(list_id, subscription_type, emails)
+    client.Lists.delete_list(list_id)
+    
+    # Add subscribers to a list, this will follow the lists double opt in settings
+    client.Lists.add_subscribers_to_list(list_id, profiles)
+        profiles: is list of objects formatted like {'email': EMAIL, 'custom_property': NAME}
+     
+    # Check email address subscription status to a list
+    client.Lists.get_list_subscription_status(list_id, emails)
+        emails: is a list of email addresses
+    
+    # Unsubscribe and remove profile from a list
+    client.Lists.delete_subscribers_from_list(list_id, emails)
+        emails: is a list of email addresses 
+
+    # Add members to a list, this doesn't care about the list double opt in setting
+    client.Lists.add_members_to_list(list_id, profiles)
+        profiles: is list of objects formatted like {'email': EMAIL, 'custom_property': NAME}
+        
+    # Check email addresses if they're in a list
+    client.Lists.get_list_membership_status(list_id_, emails)
+        emails: is a list of email addresses
+     
+    # Remove emails from a list
+    client.Lists.delete_list_membership_status(list_id, emails)
+        emails:  a list of email addresses
     
     # get exclusion emails from a list - marker is used for paginating
-    client.list_exclusions(list_id, marker=None)
+    client.Lists.get_list_exclusions(list_id, marker=None)
     
     # get all members in a group or list
-    client.all_members(group_id)
+    client.Lists.get_all_members(group_id, marker=None)
     
 You can fetch profile information given the profile ID
 
     # get profile by profile_id
-    client.get_profile(profile_id)
+    client.Profiles.get_profile(profile_id)
     
     # get all metrics for a profile with the default kwargs
     # to paginate the responses you will get a UUID returned from the response, see here for more information
     # https://www.klaviyo.com/docs/api/people#metrics-timeline
-    client.get_profile_metrics_timeline(profile_id, since=None, count=100, sort='desc')
+    client.Profiles.get_profile_metrics_timeline(profile_id, since=None, count=100, sort='desc')
 
     # get all metrics for a profile with the default kwargs
     # to paginate the responses you will get a UUID returned from the response, see here for more information
     # https://www.klaviyo.com/docs/api/people#metrics-timeline
-    client.get_profile_metric_timeline(profile_id, metric_id, since=None, count=100, sort='desc')
+    client.Profiles.get_profile_metrics_timeline_by_id(profile_id, metric_id, since=None, count=100, sort='desc')
+
+## Rate Limiting
+  If a rate limit happens it will throw a klaviyo.exceptions.KlaviyoRateLimitException
+  This will contain a detail key with a string value mentioning the time to back off in
```

Comment:
is this supposed to say "time to back off in seconds" or is that the end of the sentence?